src/marshmallow/utils.py

```diff
@@ -161,17 +161,28 @@ def from_iso_datetime(value):
     return dt.datetime(**kw)
 
 
-def from_iso_time(value):
+def from_iso_time(value, format: typing.Optional[str] = None):
     """Parse a string and return a datetime.time.
 
     This function doesn't support time zone offsets.
     """
+    invalid_format_value_error = ValueError("Not a valid ISO8601-formatted time string")
+
+    if format:
+        try:
+            return dt.datetime.strptime(value, format).time()
```

Comment:
suggestion: Maybe this strptime logic should go in `fields.Time`?

That would keep things consistent with `fields.DateTime`.